packages/runtime/fixtures/custom-metrics/external/index.mjs

```diff
@@ -0,0 +1,43 @@
+import fastify from 'fastify'
+
+globalThis.platformatic.registerMetrics((client, registry) => {
```

Comment:
I would prefer a slighlty different API and invert this: `globalThis.platformatic.getPromRegistry()`.